dawn/renderer/components/peripherals/BooleanSensor.js

```diff
@@ -1,51 +1,42 @@
+/* eslint-disable camelcase */
```

Comment:
Why is this necessary?

Reply:
It flags on the `device_name`, `device_type`, etc variables when unpacking from the object.